JobConfig/digitize/Extracted.fcl

```diff
@@ -21,18 +21,5 @@ physics.filters.Triggerable.MaxParticleMom : 1e10 # allow 'infintie' momentum co
 # extracted geometry
 services.GeometryService.inputFile: "Offline/Mu2eG4/geom/geom_common_extracted.txt"
 services.GeometryService.bFieldFile: "Offline/Mu2eG4/geom/bfgeom_no_field.txt"
-# setup CRV lookup tables to follow
-physics.producers.CrvPhotons.CRVSectors : ["EX","T1","T2"] # used only to match the vector entries below
-physics.producers.CrvPhotons.reflectors : [  0 ,  1 ,  0 ]
-physics.producers.CrvPhotons.lookupTableFileNames : ["CRVConditions/v6_0/LookupTable_6000_0",  # EX
-  "CRVConditions/v6_0/LookupTable_2370_1",  # T1
-  "CRVConditions/v6_0/LookupTable_3200_0"]  # T2
-# number of photons per MeV visible energy deposited
-# for 68 PE/SiPM @ 1 m away from SiPM (Test beam June 2017)
-# using the pulse height calibration, this value gives 45 PE/SiPM @ 1 m away from SiPM
-physics.producers.CrvPhotons.scintillationYields  : [39400,39400,39400]
-physics.producers.CrvCoincidence.CRVSectors   : ["EX","T1","T2"]
-physics.producers.CrvCoincidence.PEthresholds : [  8 ,  8 ,  8 ]
-physics.producers.CrvCoincidence.adjacentPulseTimeDifferences  : [ 10 , 10 , 10 ]
-physics.producers.CrvCoincidence.maxTimeDifferences            : [ 10 , 10 , 10 ]
-physics.producers.CrvCoincidence.coincidenceLayers             : [  3 ,  3 ,  3 ]
```

Comment:
I compared the content of the new file to the lines removed in this file.

There is one definition in the included file that's not present among the removed lines:
physics.producers.CrvPhotons.crvStepModuleLabels  : ["CrvSteps"]

There are 4 definitions in the removed lines that are not present in the included file:
physics.producers.CrvCoincidence.PEthresholds : [  8 ,  8 ,  8 ]
physics.producers.CrvCoincidence.adjacentPulseTimeDifferences  : [ 10 , 10 , 10 ]
physics.producers.CrvCoincidence.maxTimeDifferences            : [ 10 , 10 , 10 ]
physics.producers.CrvCoincidence.coincidenceLayers             : [  3 ,  3 ,  3 ]

Is this an issue?

Reply:
The 4 lines that are not present in the included file refer to an old module that is no longer used. This part is correct.
I need to double check the line that was not present in the removed line.

Reply:
@kutschke You are right, this line and another in https://github.com/Mu2e/Offline/blob/41b659c9aae8f944721c808f328783afa7ab3350/CRVResponse/fcl/epilog_extracted_v03.fcl#L1-L2 
should not be part of the included epilog, because Production needs to decide what the names of the collections are. These values are set in https://github.com/Mu2e/Production/blob/28f21e30d34ba55f3a712f49763840f9b9fc3269/JobConfig/digitize/epilog.fcl#L8-L9
I will make another PR for Offline that should be merged first.

Reply:
The Offline PR is 1223.